packages/ui-components/src/components/text-field/text-field.scss

```diff
@@ -0,0 +1,98 @@
+@import '../../assets/styles/globals';
+
+$defaultInputHeight: 35px;
```

Comment:
Use the same strategy used in the `<kv-loader>`
```suggestion
$defaultInputHeight: var(--input-height, 35px);
```

Reply:
Wouldn't it be better to wrap this CSS property `--input-height` in a `:host` selector so it can be configured from outside. [(source)](https://stenciljs.com/docs/styling#css-custom-properties)